<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cambio</class>
 <widget class="QDialog" name="cambio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="fondo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/files/fondos/fondocambiopok.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pok3">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>150</y>
     <width>301</width>
     <height>155</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/files/combate/poknoselec.png);
background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pok2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>17</y>
     <width>301</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/files/combate/poknoselec.png);
background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pok1">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-5</y>
     <width>301</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border-image: url(:/files/combate/poknoselec.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="min1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>55</y>
     <width>60</width>
     <height>53</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;

border-image: url(:/files/pokemon/charizard.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="min2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>75</y>
     <width>60</width>
     <height>53</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;

border-image: url(:/files/combate/vs.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="min3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>205</y>
     <width>60</width>
     <height>53</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;

border-image: url(:/files/combate/vs.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="est1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;

border-image: url(:/files/estados/CONGELADO.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="est2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;


border-image: url(:/files/estados/CONGELADO.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="est3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>180</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;


border-image: url(:/files/estados/CONGELADO.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="v1">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>100</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QLabel" name="v11">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>100</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="v2">
   <property name="geometry">
    <rect>
     <x>498</x>
     <y>121</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QLabel" name="v22">
   <property name="geometry">
    <rect>
     <x>421</x>
     <y>121</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="v3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QLabel" name="v33">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>250</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="pok4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>301</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/files/combate/poknoselec.png);
background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="n1">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>101</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>80</string>
   </property>
  </widget>
  <widget class="QLabel" name="n2">
   <property name="geometry">
    <rect>
     <x>343</x>
     <y>123</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>80</string>
   </property>
  </widget>
  <widget class="QLabel" name="n3">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>251</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>80</string>
   </property>
  </widget>
  <widget class="QLabel" name="n4">
   <property name="geometry">
    <rect>
     <x>343</x>
     <y>276</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>80</string>
   </property>
  </widget>
  <widget class="QLabel" name="est4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;


border-image: url(:/files/estados/CONGELADO.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="v44">
   <property name="geometry">
    <rect>
     <x>421</x>
     <y>275</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="vida4">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>240</y>
     <width>115</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border-image: url(:/files/combate/verde.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="min4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>225</y>
     <width>60</width>
     <height>53</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;

border-image: url(:/files/combate/vs.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pok5">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>300</y>
     <width>301</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/files/combate/poknoselec.png);
background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pok6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>301</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/files/combate/poknoselec.png);
background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="min5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>355</y>
     <width>60</width>
     <height>53</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;

border-image: url(:/files/combate/vs.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="min6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>375</y>
     <width>60</width>
     <height>53</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;

border-image: url(:/files/combate/vs.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="v4">
   <property name="geometry">
    <rect>
     <x>498</x>
     <y>275</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QLabel" name="v6">
   <property name="geometry">
    <rect>
     <x>498</x>
     <y>424</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QLabel" name="v5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>405</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QLabel" name="v55">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>405</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="v66">
   <property name="geometry">
    <rect>
     <x>421</x>
     <y>424</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="vida6">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>390</y>
     <width>115</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border-image: url(:/files/combate/verde.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="vida2">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>90</y>
     <width>115</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border-image: url(:/files/combate/verde.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="vida1">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>70</y>
     <width>115</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border-image: url(:/files/combate/verde.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="vida3">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>220</y>
     <width>115</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border-image: url(:/files/combate/verde.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="vida5">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>370</y>
     <width>115</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border-image: url(:/files/combate/verde.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="n5">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>406</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>80</string>
   </property>
  </widget>
  <widget class="QLabel" name="n6">
   <property name="geometry">
    <rect>
     <x>343</x>
     <y>426</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>80</string>
   </property>
  </widget>
  <widget class="QLabel" name="est6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>350</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;


border-image: url(:/files/estados/CONGELADO.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="est5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;


border-image: url(:/files/estados/CONGELADO.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nombre1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>Abomasnow</string>
   </property>
  </widget>
  <widget class="QLabel" name="nombre3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>Abomasnow</string>
   </property>
  </widget>
  <widget class="QLabel" name="nombre5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>Abomasnow</string>
   </property>
  </widget>
  <widget class="QLabel" name="nombre2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>Abomasnow</string>
   </property>
  </widget>
  <widget class="QLabel" name="nombre4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>Abomasnow</string>
   </property>
  </widget>
  <widget class="QLabel" name="nombre6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>350</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Pokemon Pinball RS</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>Abomasnow</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>281</width>
     <height>135</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>281</width>
     <height>135</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>fondo</zorder>
  <zorder>vida6</zorder>
  <zorder>vida5</zorder>
  <zorder>vida4</zorder>
  <zorder>vida3</zorder>
  <zorder>vida2</zorder>
  <zorder>vida1</zorder>
  <zorder>pok3</zorder>
  <zorder>pok2</zorder>
  <zorder>pok1</zorder>
  <zorder>min1</zorder>
  <zorder>min2</zorder>
  <zorder>min3</zorder>
  <zorder>est1</zorder>
  <zorder>est2</zorder>
  <zorder>est3</zorder>
  <zorder>v1</zorder>
  <zorder>v11</zorder>
  <zorder>v2</zorder>
  <zorder>v22</zorder>
  <zorder>v3</zorder>
  <zorder>v33</zorder>
  <zorder>pok4</zorder>
  <zorder>n1</zorder>
  <zorder>n2</zorder>
  <zorder>n3</zorder>
  <zorder>n4</zorder>
  <zorder>est4</zorder>
  <zorder>v44</zorder>
  <zorder>min4</zorder>
  <zorder>pok5</zorder>
  <zorder>pok6</zorder>
  <zorder>min5</zorder>
  <zorder>min6</zorder>
  <zorder>v4</zorder>
  <zorder>v6</zorder>
  <zorder>v5</zorder>
  <zorder>v55</zorder>
  <zorder>v66</zorder>
  <zorder>n5</zorder>
  <zorder>n6</zorder>
  <zorder>est6</zorder>
  <zorder>est5</zorder>
  <zorder>nombre1</zorder>
  <zorder>nombre3</zorder>
  <zorder>nombre5</zorder>
  <zorder>nombre2</zorder>
  <zorder>nombre4</zorder>
  <zorder>nombre6</zorder>
  <zorder>btn1</zorder>
  <zorder>btn2</zorder>
  <zorder>btn3</zorder>
  <zorder>btn4</zorder>
  <zorder>btn6</zorder>
  <zorder>btn5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
